<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SepecificSetting</class>
 <widget class="QWidget" name="SepecificSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>34</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLineEdit" name="SpecificSettingName"/>
   </item>
   <item>
    <widget class="QSpinBox" name="SpecificSettingNumber"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
